### PR TITLE
Add see more button to 2020 mentee list

### LINKED ISSUE #1490: Reduce length of the 2020 scholarX mentee list by adding a see more button
**Describe the bug**
The mentee list on the scholarX 2020 page is a bit lengthy

**To Reproduce**
Create a see more button for the mentee list so that we can reduce the length of the page without showing all the mentees at once.

**Expected behavior**
There should be a see more button to show all the mentees when clicked

**Screenshots**
Screenshot from 2023-01-10 10-47-36

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by gajindubandara**
## Purpose

The purpose of this PR is to fix #1490

## Goals

Add a see more button to the mentee section to reduce the length of the page.

## Approach

Added a see more button so when the button is clicked once it will display six more profiles. This will continue until all the profiles are displayed. There is a see-less button as well. 

### Screenshots

Initially, the see more button is visible 
  Screenshot from 2023-01-10 11-54-55

When the see more button is once clicked the see less button will appear
Screenshot from 2023-01-10 11-55-03

When all the profiles are displayed the see more button will disappear
Screenshot from 2023-01-10 11-55-22
### Preview Link

https://pr-1491-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/past-programs/2020/index.html [hunk 1]
```diff
@@ -446,12 +450,57 @@ <h5>Projects :</h5>
...
+                            $('#templateMentees').html(), { 'data': nextPart });
+                        $(nextProfileSet).appendTo('#teamMentees').hide().fadeIn(1000);
+                        $("#mentee-show-less").show();
+
+                        if(tempProfileCount>=profileCount){
+                            $('#mentee-show-more').hide();
+                        }
+                    });
+
+                    $("#mentee-show-less").click(function () {
```
Comment (anjula-sack) [inline scholarx/past-programs/2020/index.html]: Do we have a similar code snippet somewhere else? @gajindubandara

Comment (gajindubandara) [reply scholarx/past-programs/2020/index.html]: A similar code is used in the `scholarx/past-programs/2019/index.html`. Shall I Make a generalized function to load mentees on the `assets/js/functions.js `  ? @anjula-sack

Comment (anjula-sack) [reply scholarx/past-programs/2020/index.html]: yes please @gajindubandara

File: assets/js/functions.js [hunk 2]
```diff
@@ -59,3 +59,62 @@ function loadPosts(url){
...
+        type: 'get',
+        url: url,
+        dataType: 'json',
+        success: function (data) {
+            document.getElementById('btnMentees').style.visibility = "visible";
+            //slice array to two parts
+            if (data.length >= 6) {
+                let tempProfileCount =6;
+                let profileCount =data.length;
+                let initialPart = data.slice(0, 6);
```
Comment (anjula-sack) [inline assets/js/functions.js]:
```suggestion
                let initialProfileCount = 6;
                let profileCount = data.length;
                let initialPart = data.slice(0, initialProfileCount);
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/past-programs/2020/index.html [hunk 3]
```diff
@@ -214,7 +214,7 @@ <h1 class="display-3 text-center mb-5">Mentors</h1>
 </section>
 
 <section class="section pb-0 bg-gradient-white">
-    <div class="container">
+    <div class="container" id="mentees_div">
```
Comment (anjula-sack) [inline scholarx/past-programs/2020/index.html]:
```suggestion
    <div class="container" id="mentees">
```

File: scholarx/past-programs/2019/index.html [hunk 4]
```diff
@@ -133,7 +133,7 @@ <h1 class="display-3 text-center mb-5">Mentors</h1>
 </section>
 
 <section class="section pb-0 bg-gradient-white">
-    <div class="container">
+    <div class="container" id="mentees_div">
```
Comment (anjula-sack) [inline scholarx/past-programs/2019/index.html]:
```suggestion
    <div class="container" id="mentees">
```

File: assets/js/functions.js [hunk 5]
```diff
@@ -59,3 +59,62 @@ function loadPosts(url){
         }
     });
 }
+//function to load mentee profiles
+function loadMentees(url) {
```
Comment (anjula-sack) [inline assets/js/functions.js]:
```suggestion
function loadMentees(sheetURL) {
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Thank you! @gajindubandara

**Pull request merged**